dashboard/src/components/WorkflowCollapse.vue

```diff
@@ -46,12 +46,21 @@ const showAipDeletionReviewAlert = (
     workflow.status == api.EnduroStorageAipWorkflowStatusEnum.Pending
   );
 };
+
+// Auto-expand the workflow task list if the workflow is "in progress".
+const wfBtn = ref<HTMLElement | null>(null);
+onMounted(() => {
+  if (workflow.value.status === "in progress") {
+    wfBtn.value?.click();
+  }
+});
```

Comment:
Feels like a hack to do this. Why not checking in the attributes directly? I think it only requires to remove the collapsed class and set aria-expanded to true in the button, and add show to the content classes. Like the first item in the first example:

https://getbootstrap.com/docs/5.3/components/accordion/

Reply:
Fine, I'll do it the "right" way. :roll_eyes: :stuck_out_tongue_closed_eyes: